src/components/page/page.jsx

```diff
@@ -1,21 +1,12 @@
-import React, { Component } from 'react'
+import React from 'react'
 
 import Spinner from 'components/spinner'
 import { connect } from 'react-redux'
 import styles from './style.scss'
 
-class Page extends Component {
-  render () {
-    const { children, isLoading } = this.props
-    return (
-      <div className={styles.page}>
-        {isLoading
-          ? <Spinner />
-          : children
-        }
-      </div>
-    )
-  }
+const Page = props => {
```

Comment:
Out of curiosity: Why not this?
```js
const Page = ({ children, isLoading }) => {
```

Reply:
I prefer to have props and extract them later, but there’s no right or wrong, it could be just like the way you mentioned.